README.md

```diff
@@ -1,2 +1,230 @@
-# coalaip-http-api
-An HTTP wrapper for CoalaIP
+# COALA IP HTTP Wrapper
+
+> A HTTP wrapper for [pycoalaip](https://github.com/bigchaindb/pycoalaip).
+
+
+## Why?
+
+[pycoalaip](https://github.com/bigchaindb/pycoalaip) is Python-specific.
+Furthermore all underlying dependencies are written in Python, which means
+that parties interested in COALA IP would have to re-implement large parts of
+their logic in their programming language to make use of COALA IP.
+To solve this problem in the short-term, this library exposes a RESTful web
+interface (runnable in Docker) of the functionalities provided in
+[pycoalaip](https://github.com/bigchaindb/pycoalaip).
+
+
+## Whats the status of this?
+
+Super-pre-alpha. At the moment all you can do is hit against two endpoints that
+are being exposed. Minimal error paths are provided.
+This package and [pycoalaip](https://github.com/bigchaindb/pycoalaip)
+will probably mature rather quickly as we're actively developing them
+currently.
+
+
+## Can I use this in production?
+
+No. Currently we're hitting against a BigchainDB instance running in
+quasi "regtest" mode. Its never been used in production either.
+In its current state, this package can and should be used to experiment with
+COALA IP.
+
+
+## OK, how can I start experimenting?
+
+1. Read the installation section on how to install with Docker.
+2. Get familiar with the REST API provided at the end of this document.
+3. Hit against a running Docker container with a HTTP client of your choice.
+
+
+## Should I expose this server to the internet?
+
+NO! Think of this library more like a shell-command. The flask server should at
+least be run behind a reverse-proxy like nginx, so that its interface is NOT
+exposed to the world wide web.
+
+
+## Can I just have a shell-command for this?
+
+Yes, [curl](https://curl.haxx.se/).
+
+
+## Installation and Usage
+
+
+### For development on Linux
+
+```
+$ git clone git@github.com:bigchaindb/coalaip-http-api.git
+$ virtualenv --python=python3 venv
+$ source venv/bin/activate
+$ pip install -r requirements_dev.txt
+$ python web/server.py
+```
```

Comment:
Updated in https://github.com/bigchaindb/coalaip-http-api/pull/2/commits/8d78073ad33be5e3c8198dfd94e2d051a8e69f01